modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v2_0/test/DataDefinitionResourceTest.java

```diff
@@ -262,6 +262,12 @@ public void testGraphQLGetSiteDataDefinitionByContentTypeByDataDefinitionKeyNotF
 				"Object/code"));
 	}
 
+	@Override
+	@Test
+	public void testPatchDataDefinition() throws Exception {
```

Comment:
@alejoceballos @rebsilva 
Do we need to override this one? i think overriding `testPatchDataDefinition_addDataDefinition()` is enough , no?

Reply:
That's true Jey, thanks!